src/MusicStore/Components/AnnouncementComponent.cs

```diff
@@ -25,6 +26,13 @@ public IMemoryCache Cache
             set;
         }
 
+        [Activate]
+        public ISystemClock SystemClock
```

Comment:
We should redefine the interface here. The one from caching is effectively an internal implementation detail.